app/products/calls/components/floating_call_container.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {View, Platform, StyleSheet} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import {ANDROID_DEFAULT_HEADER_HEIGHT, IOS_DEFAULT_HEADER_HEIGHT} from '@constants/view';
+
+let topBarHeight = ANDROID_DEFAULT_HEADER_HEIGHT;
+if (Platform.OS === 'ios') {
+    topBarHeight = IOS_DEFAULT_HEADER_HEIGHT;
+}
+
+const style = StyleSheet.create({
+    wrapper: {
+        position: 'absolute',
+        width: '100%',
+        ...Platform.select({
+            android: {
+                elevation: 9,
```

Comment:
elevation does create a shadow, is that what you want here? if so, why not apply the shadow on iOS?